docs/src/main/paradox/cleanup.md

```diff
@@ -0,0 +1,30 @@
+# Database Cleanup
```

Comment:
Good with a separate page. Also link to this from journal.md "Delete all events", snapshot.md "Delete all snapshots" and events-by-tag.md "Cleanup of tag_views table"

Reply:
👍